addons/website_event_track/views/event_track_views.xml

```diff
@@ -118,12 +118,13 @@
                             <field name="website_url" invisible="1"/>
                             <field name="is_published" widget="website_redirect_button"/>
                         </div>
-                        <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
-                        <field name="kanban_state" widget="state_selection"/>
+                        <widget name="web_ribbon" text="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
+                        <field name="image" widget="image" class="oe_avatar"/>
                         <div class="oe_title">
                             <label for="name" class="oe_edit_only"/>
                             <h1>
                                 <field name="name" placeholder="e.g. Inspiring Business Talk"/>
+                                <field name="kanban_state" widget="state_selection" class="ml-3"/>
```

Comment:
Are we sure about moving that button there?
Didn't see anything about it in the task pad.